public/src/unityIAP/java/com/revenuecat/purchases/models/BC4StoreProduct.kt

```diff
@@ -0,0 +1,60 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import com.android.billingclient.api.SkuDetails
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.ComparableData
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.parceler.JSONObjectParceler
+import kotlinx.parcelize.Parcelize
+import kotlinx.parcelize.RawValue
+import kotlinx.parcelize.TypeParceler
+import org.json.JSONObject
+
+@Parcelize
+@TypeParceler<JSONObject, JSONObjectParceler>()
```

Comment:
I think we should make this also implement storeProduct in the long run, but we can address later

Reply:
i believe it already does!